code/src/components/app-typography/AppTypography.tsx

```diff
@@ -0,0 +1,52 @@
+import { FormattedMessage } from "react-intl";
+import "./AppTypography.scss";
+import Typography, { TypographyProps } from "@mui/material/Typography";
+import { ReactNode } from "react";
```

Comment:
organize imports and use alias for importing scss

Reply:
Fixed